mobile-app-automation/readme.md

```diff
@@ -0,0 +1,82 @@
+# Coding task for Mobile QA Automation Engineer @tradeling
+
+## The Goal
+
+Your task is to write automated tests preferably with [Appium](https://appium.io/), or [Detox](https://github.com/wix/Detox/)  to demonstrate your test automation abilities.
+
+
+## The Process
+
+1. Start a new github repository with detailed "README" document on how to execute the tests.
+1. Download Amazon Shopping Android or App Store Package (APK or IPA)
+1. Write end-to-end tests fulfilling the [Acceptance Criteria](#acceptance-criteria) below.
+1. Publish test execution report in html format.
+1. Prepare a test summary report and include the information about platform, app, tools etc
+1. Include all tests related document in same github repository.
+
+- Use below package and activity name (Appium)
+  - appPackage=com.amazon.mShop.android.shopping
+  - appActivity=com.amazon.mShop.home.HomeActivity
+
+## Acceptance Criteria
+
+- Scenario 1: [Check Add a new Payment to your profile]
+   1. Starts on the [Amazon](https://www.amazon.com/)
+   2. Login with valid credentials
+   3. Tap on Your account
+   4. Tap on the Payment option
+   5. Tap on Wallet and add a credit or debit card as the payment method
+
+
+- Scenario 2: [Check your gift card's balance is $0.00]
+   1. Tap on Your account
+   2. Tap on Gift Cards
+   3. Check the balance of your gift card is $0.00
+
+
+- Scenario 3: [Check the total displayed number of results for category Smart Home | Televisions]
+   1. Go to Categories list
+   2. Choose category 'Smart Home'
+   3. Choose sub-category category 'Home Entertainment'
+   4. Choose sub-category 'Televisions'
+   5. Check the total number of results match the total displayed products
```

Comment:
I believe, all steps needs to be revisit as mobile app doesn't have the same like web, thanks

Reply:
@MohmedEmad I added it as is, only after when I received the tasks after internal review among the QA team.

@Elena0420  can you please give us the exact steps as I tried to implement what I have received.

Reply:
@PrinceSoni83 I will provide you with the detailed steps Today, thanks